src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: $card-width;
+  margin-bottom: 46px;
+  margin-right: 48px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__icon {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-size: $medium-size-text;
+    font-weight: $normal-weight;
+    line-height: 18px;
+    color: $emphasized-color;
+  }
+
+  &__productCode {
+    margin-bottom: $margin-bottom;
+
+    font-size: $small-size-text;
+    line-height: 14px;
+    color: $lightened-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+
+    line-height: 14px;
+  }
+
+  &__review-mark {
+    font-size: $small-size-text;
+    color: $emphasized-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: $margin-bottom;
+  }
+
+  &__price-text {
+    font-size: $medium-size-text;
+    color: $lightened-color;
+  }
+
+  &__price-sum {
+    font-size: 16px;
+    font-weight: $bold-weight;
+    line-height: 18px;
+    color: $emphasized-color;
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    padding: 11px 0;
+
+    font-size: 14px;
+    font-weight: $bold-weight;
+    color: $white-color;
+
+    background-color: $button-color;
+    border: 1px solid $button-color;
+    border-radius: 5px;
+
+    @include transition(color 0.3s);
```

Comment:
Good, but you can just pass property, no need to set duration every time, you can leave it in mixin